client/state/login/actions.js

```diff
@@ -93,14 +95,22 @@ const errorFields = {
 	invalid_username: 'usernameOrEmail',
 };
 
+function getLocalizedLoginURL( action ) {
+	if ( 'en' === i18nUtils.getLocaleSlug() ) {
+		return 'https://wordpress.com/wp-login.php?action=' + action;
+	}
+
+	return 'https://' + i18nUtils.getLocaleSlug() + '.wordpress.com/wp-login.php?action=' + action;
+}
+
 /**
  * Retrieves the first error message from the specified HTTP error.
  *
  * @param {Object} httpError HTTP error
  * @returns {{code: string?, message: string, field: string}} an error message and the id of the corresponding field, if not global
  */
 function getErrorFromHTTPError( httpError ) {
-	let message;
+	let message, code;
```

Comment:
Super minor but we don't really have to declare `code` here. This can happen when we actually use it:

```
let code = get( httpError, 'response.body.data.errors.code' );
```